### DISCUSSION TITLE
Return to Castle Wolfenstein

### DISCUSSION THREAD

**Discussion by ferrellsl** [Joshua-Riek/ubuntu-rockchip#96] [General]
Just completed a successful build of Return to Castle Wolfenstein.  It's a very simple process.  Just follow the directions here: 
https://github.com/iortcw/iortcw

After cloning the repo, just enter the SP directory for the single-player version and type:  make
For multi-player, enter the MP directory and type:  make

Copy your existing PAK files from your PC to your PI 5.  The PAK files on your PC are located in the “Main” folder. Copy the following files over to your iortcw's “Main” folder for the single player campaign: pak0.pk3, sp_pak1.pk3, sp_pak2.pk3, and sp_pak3.pk3. For multiplayer, copy the following files to your iortcw's “Main” folder: mp_bin.pk3, mp_pak0.pk3, mp_pak1.pk3, mp_pak2.pk3, mp_pak3.pk3, mp_pak4.pk3, mp_pak5.pk3, mp_pakmaps0.pk3, mp_pakmaps1.pk3, mp_pakmaps2.pk3, mp_pakmaps3.pk3, mp_pakmaps4.pk3, mp_pakmaps5.pk3, and mp_pakmaps6.pk3

I'm using the non-Steam version PAK files so I'm not sure if the PAK files from Steam will work or not.

**Comment by ai760** [comment]
Looking for more .pak files
